starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,72 @@
-// Names and Input
+let hacker1  = "will";
+let hacker2 =prompt('What is your name?');
+
+
+
+if (hacker1.length == hacker2.length){
+
+    console.log("wow, you both got equally long names," +" "+ hacker1.length +" "+ "characters!!");
+    
+}
+      else if(hacker2.length > hacker1.length){
+        console.log("Yo, navigator got the longest name, it has" +" "+ hacker2.length +" "+ "characters");
+    } 
+    
+    else if(hacker1.length > hacker2.length) {
+        console.log("The Driver has the longest name, it has"+  hacker1.length +"characters");
```

Comment:
Not a  huge deal, but don't forget to add spacing between the strings and values that you're concatenating (to improve readability). For example, if hacker1 was the longest name, it would print:

 "The Driver has the longest name, it has4characters"